design/KruizePromQL.md

```diff
@@ -81,29 +93,39 @@ The following are the available Kruize DB methods that you can monitor:
 - `addPerformanceProfileToDB`: Method to add performance profile to the database.
 - `loadPerformanceProfileByName`: Method to load a specific performance profile.
 - `loadAllPerformanceProfiles`: Method to load all performance profiles.
+- `genratePlots`: methode to generate plots for all terms.
```

Comment:
It should be `generatePlots`. `method` is wrongly spelled as well

Reply:
Please change it to: `Method to generate blox plot metrics for all terms`

Reply:
done